extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -266,14 +266,14 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 		this._instanceProperties.dataController = config?.metadata.name || this._instanceProperties.dataController;
 		this._instanceProperties.region = this._azurecoreApi.getRegionDisplayName(config?.spec.settings.azure.location) || this._instanceProperties.region;
 		this._instanceProperties.subscriptionId = config?.spec.settings.azure.subscription || this._instanceProperties.subscriptionId;
-		// this._instanceProperties.vCores = reg.vCores || '';
 		this.refreshDisplayedProperties();
 	}
 
 	private handleMiaaConfigUpdated(): void {
 		if (this._miaaModel.config) {
 			this._instanceProperties.status = this._miaaModel.config.status.state || '-';
 			this._instanceProperties.externalEndpoint = this._miaaModel.config.status.externalEndpoint || loc.notConfigured;
+			this._instanceProperties.vCores = this._miaaModel.config.spec.limits?.vcores?.toString() || '';
```

Comment:
change 
**_....vcores?.toString() || '';_**
to:
**_....vcores?.toString() ?? '';_**

The current version of the code will convert 0 (zero) vCores to an empty string which is not what we are intending. This change is good to do for other lines above as well.

Reply:
https://github.com/microsoft/azuredatastudio/blob/main/extensions/arc/src/localizedConstants.ts#L129 it actually is fine and doesn't matter in this particular case.